src/app/store/submitted-transactions/submitted-transactions.selectors.ts

```diff
@@ -0,0 +1,11 @@
+import { RootState, store } from '@app/store';
+
+import { submittedTransactionsAdapter } from './submitted-transactions.slice';
+
+const submittedTransactionsSelectors = submittedTransactionsAdapter.getSelectors<RootState>(
+  state => state.submittedTransactions
+);
+
+export function useSubmittedTransactions() {
+  return submittedTransactionsSelectors.selectAll(store.getState());
```

Comment:
```suggestion
  return useSelector(submittedTransactionsSelectors.selectAll);
```

Should use `useSelector` to read from store. 

Selectors "subscribe" to the store, so that future updates trigger renders. Here, it's different in that we do a one-time read from the store at render-time and return the value. It probably works, as renders happen often, but is subtly different and could be v confusing if, say, a render wasn't triggered.